app/views/docs/authentication.phtml

```diff
@@ -691,76 +889,649 @@ let session = try await account.get()</code></pre>
+    .setProject('5df5acd0d48c2');               // Your project ID
+
+const account = new Account(client);
+
+const promise = account.updateRecovery('[USER_ID]', '[SECRET]', 'password', 'password');
+
+promise.then(function (response) {
+    console.log(response);
+}, function (error) {
+    console.log(error);
+});</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2');               // Your project ID
+
+final account = Account(client);
+
+final user = await account.updateRecovery(
+    userId: '[USER_ID]',
+    secret: '[SECRET]',
+    password: 'password'
+    passwordAgain: 'password'
+);</code></pre>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+val account = Account(client)
+
+val token = account.updateRecovery(
+    userId = "[USER_ID]",
+    secret = "[SECRET]",
+    password = "password",
+    passwordAgain = "password"
+)</code></pre>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let account = Account(client)
+
+let token = try await account.updateRecovery(
+    userId: "[USER_ID]",
+    secret: "[SECRET]",
+    password: "password",
+    passwordAgain: "password"
+)</code></pre>
+    </li>
+    <li>
+        <h3>GraphQL</h3>
+        <div class="ide" data-lang="graphql" data-lang-label="GraphQL">
+            <pre class="line-numbers"><code class="prism language-graphql" data-prism>mutation {
+    accountUpdateRecovery(
+        userId: "[USER_ID]",
+        secret: "[SECRET]",
+        password: "password",
+        passwordAgain: "password"
+    ) {
+        _id
+        _createdAt
+        userId
+        secret
+        expire
+    }
+}</code></pre>
+        </div>
+    </li>
+</ul>
+
+
+<h2><a href="/docs/authentication#jwt" id="jwt">JWT Authentication</a></h2>
+<p>If you're building server applications using Appwrite's Server SDKs, you can use JWTs for authentication between your client and server applications.</p>
```

Comment:
Would it be beneficial to have a single line explaining what JWT is in the opening of this section? it would also be great to have an external link to a relevant JWT resource (official website / wikipedia). Please make sure to make the link open in new tab and have nofollow, noindex attributes.

Reply:
😬 This is getting, looonnggg, but okay,

Reply:
<img width="949" alt="Screenshot 2023-01-30 at 6 11 20 PM" src="https://user-images.githubusercontent.com/29069505/215617497-f12bd09b-8d59-46f0-b099-d7eb12a69ccf.png">